cmd/api/book/service.go

```diff
@@ -1,27 +1,31 @@
 package book
 
 import (
+	"context"
+	"errors"
 	"math"
 	"time"
 
 	"github.com/google/uuid"
 )
 
+var RequestTimeout = time.Duration(5) * time.Second
```

Comment:
This should not be a global variable. You need to add this to the `http.BookHandler` struct, its part of the service configuration.

Reply:
Remember, dependency injection.